libs/gl-client/src/credentials.rs

```diff
@@ -291,8 +291,32 @@ mod model {
     }
 }
 
+/// Tries to load nobody credentials from a file that is passed by an envvar and
+/// defaults to the nobody cert and key paths that have been set during build-
+/// time.
+fn load_file_or_default(varname: &str, default: &[u8]) -> Vec<u8> {
+    match std::env::var(varname) {
+        Ok(fname) => {
+            debug!("Loading file {} for envvar {}", fname, varname);
+            let f = std::fs::read(fname.clone());
+            if f.is_err() {
+                debug!(
+                    "Could not find file {} for var {}, loading from default",
+                    fname, varname
+                );
```

Comment:
Should we really allow this to go back to default almost quietly? Failing loudly here is likely cleaner.

Reply:
Would you prefer to panic or do you think raising the log level is enough?